From 1e9a8f14ff52f26b3d39bbc5716ac51f1f34c33c Mon Sep 17 00:00:00 2001
From: Kevin Wang <kevin1.wang@amd.com>
Date: Thu, 2 Jan 2020 13:59:17 +0800
Subject: [PATCH 1261/1453] drm/amdgpu: use linux size macro to simplify
 ONE_Kib & One_Mib

replace internal size macro with linux size macro

Signed-off-by: Kevin Wang <kevin1.wang@amd.com>
Reviewed-by: Tianci Yin <tianci.yin@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c | 6 +++---
 drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.h | 7 -------
 2 files changed, 3 insertions(+), 10 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
index 98c33feb1877..e2a4bc7ff012 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
@@ -1875,10 +1875,10 @@ static int amdgpu_ttm_training_reserve_vram_fini(struct amdgpu_device *adev)
 
 static u64 amdgpu_ttm_training_get_c2p_offset(u64 vram_size)
 {
-       if ((vram_size & (ONE_MiB - 1)) < (4 * ONE_KiB + 1) )
-               vram_size -= ONE_MiB;
+       if ((vram_size & (SZ_1M - 1)) < (SZ_4K + 1) )
+               vram_size -= SZ_1M;
 
-       return ALIGN(vram_size, ONE_MiB);
+       return ALIGN(vram_size, SZ_1M);
 }
 
 /**
diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.h
index c3eef491ae0f..c115b08529dd 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.h
@@ -70,13 +70,6 @@ struct amdgpu_copy_mem {
 	unsigned long			offset;
 };
 
-/* Definitions for constance */
-enum amdgpu_internal_constants
-{
-	ONE_KiB	= 0x400,
-	ONE_MiB	= 0x100000,
-};
-
 extern const struct ttm_mem_type_manager_func amdgpu_gtt_mgr_func;
 extern const struct ttm_mem_type_manager_func amdgpu_vram_mgr_func;
 
-- 
2.17.1

